### PR TITLE
Faster PNG rendering + class improvements

### PR DESCRIPTION
**Pull request by jmmelko**
This new beta has faster PNG rendering thanks to a new chord image.
Note class was improved, by resurrecting one of Tracey's old methods.
Some minor changes.

No rush. But I'd like to pull this before I start to tackle #21.

### CODE REVIEW

File: python/notes.py [hunk 1]
```diff
@@ -196,14 +200,15 @@ def get_png(self, highlighted_frames):
                 else:
                     return None
             else:
-                return Image.open(self.circle_highlighted_pngs[min(highlighted_frames[0]-1,len(self.circle_highlighted_pngs))])
+                return Image.open(self.circle_highlighted_pngs[min(highlighted_frames[0],len(self.circle_highlighted_pngs))-1])
```
Comment (t1-tracey) [inline python/notes.py]: there is merge conflict with these three, for the hotfix

Comment (t1-tracey) [reply python/notes.py]: the current code in beta is:

```
return Image.open(self.root_highlighted_pngs[min(highlighted_frames[0],len(self.root_highlighted_pngs))-1])
```

and the current code in master is:
```
return Image.open(self.root_highlighted_pngs[min(highlighted_frames[0]-1,len(self.root_highlighted_pngs)-1)])
>>>>>>> master
```

is the second one the intended one after the hot fix? https://github.com/sky-music/sky-python-music-sheet-maker/commit/febb7d760c3be2240f0df4d77d20df7a6b2865b1

**Comment by jmmelko** [comment]
Both lead to the same result.
The second one (beta) is more compact.
So let’s opt for the 2nd one.

**Comment by t1-tracey** [comment]
Ok 👍

Sent from my iPad

On 17 Nov 2019, at 8:43 pm, jmmelko <notifications@github.com> wrote:

﻿

Both lead to the same result.
The second one (beta) is more compact.
So let’s opt for the 2nd one.

**Comment by t1-tracey** [comment]
I am having trouble finding my tests in the list of `/songs` 😅 is it ok if I remove the default songs? (I saved them in another folder on my computer)

screenshot

**Comment by jmmelko** [comment]
Yes of course.

Actually I’ve seen that you have a side GitHub repos for songs, so I’d be happy to joint it and contribute.

JMM.

**Comment by t1-tracey** [comment]
Sure, I have set it so you have access to all of them now!

**Comment by t1-tracey** [comment]
here were the instructions I had written for my friend Sera, to upload songs to the site. (just updated it now)

https://sky.bloomexperiment.com/t/sky-music-sheets-site/13/2?u=cactusflower

**Pull request merged**